install_prerequisites.py

```diff
@@ -155,6 +150,9 @@ def main():
             'zlib-devel',
         ]
 
+        package_list.extend([
+        ])
```

Comment:
Is this needed?

Reply:
Nope, not sure where this came from either. Removed.